entity-framework/core/modeling/relationships.md

```diff
@@ -267,6 +267,55 @@ With this configuration the columns corresponding to `ShippingAddress` will be m
 
 ### Many-to-many
 
-Many-to-many relationships without an entity class to represent the join table are not yet supported. However, you can represent a many-to-many relationship by including an entity class for the join table and mapping two separate one-to-many relationships.
+Many to many relationships require a collection navigation property on both sides. They will be discovered by convention like other types of relationships.
+
+[!code-csharp[Main](../../../samples/core/Modeling/FluentAPI/Relationships/ManyToManyShared.cs?name=ManyToManyShared)]
+
+The way this relationship is implemented in the database is by a join table that contains foreign keys to both `Post` and `Tag`. This is what EF will create in SQL Server for the above model.
```

Comment:
Consider using non-SqlServer or example. This sentence make it look like it only works this way in SqlServer.